jinja2-compat/src/jinja2/__init__.py

```diff
@@ -0,0 +1,31 @@
+def _export_jinja(name, ns):
+    import re
+    import warnings
+    from importlib import import_module
+
+    mod_name = "jinja" if not name else ("jinja." + name)
+    warnings.warn(
+        "'jinja2' has been renamed to 'jinja'. Import from %r instead." % mod_name,
+        DeprecationWarning,
+        stacklevel=3,
+    )
+
+    try:
+        mod = import_module(mod_name)
+    except ImportError:
+        warnings.warn(
+            "%r no longer exists. Either fully migrate to Jinja >= 3.0"
+            " or downgrade to an earlier version." % mod_name,
+            stacklevel=3,
+        )
+        return
+
+    for key, value in vars(mod).items():
+        if re.match(r".$|_?[^_]", key):
```

Comment:
We may need a special case to allow `__version__` here. As it is, at least in Ansible, we have some version comparisons, and the shim is omitting `__version__` here.

I've debated about having an explicit `__version__` here, but that could be misleading, since the proxied version could differ I suppose.

Reply:
Should be fixed now, `__version__` is exported.